frontend/app/components/redesign/components/revshare/ShareInput.tsx

```diff
@@ -224,26 +218,13 @@ export const ShareInput = React.memo(
             Weight value for calculating revenue share.
           </div>
         </div>
-        <div role="cell" aria-labelledby="col-percent">
-          <label htmlFor={percentInputId} className="sr-only">
-            Percentage
-          </label>
-          <InputField
+        <div role="cell" aria-labelledby="col-percentage">
+          <div
             id={percentInputId}
-            type="number"
-            min={0}
-            max={100}
-            step="any"
-            value={typeof percent === 'number' ? Math.round(percent * 100) : ''}
-            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-              onChangePercent(Number(e.target.value) / 100)
-            }
-            disabled={percentDisabled}
-            aria-disabled={percentDisabled}
-            aria-describedby={`percent-description-${index}`}
-          />
-          <div id={`percent-description-${index}`} className="sr-only">
-            Calculated from weight, or can be edited directly.
+            aria-label={`Calculated percentage: ${Math.round(percent * 100)}%`}
```

Comment:
As this div has no assigned role, `aria-label` will not be used (except when using VoiceOver).
https://benmyers.dev/blog/dont-use-aria-label-on-static-text-elements/

Consider using `sr-only` and `aria-hidden` if you really want to show "calculated percentage: {}", which I think is redundant, as we're in a table and the cell has a header.

Reply:
What I mean is, there's an aria-label on a plain div here, which isn't right.